### ISSUE TITLE
FieldError at /django-admin/django_mailbox/message/ on latest release.

### ISSUE THREAD

**Issue by xeroticikot** [coddingtonbear/django-mailbox#287]
Just installed the latest release with latest django version. Getting the following error on the django admin - 
```
FieldError at /django-admin/django_mailbox/message/

Cannot resolve keyword 'attachements' into field. Choices are: attachments, body, eml, encoded, from_header, id, in_reply_to, in_reply_to_id, mailbox, mailbox_id, message_id, outgoing, processed, read, replies, subject, to_header
```

I believe the following line's causing the error - 

```
class MessageAdmin(admin.ModelAdmin):
    def get_queryset(self, *args, **kwargs):
        return super().get_queryset(*args, **kwargs).annotate(num_attachments=Count('attachements'))
```

attachements should be attachments.

**Comment by Pietro395** [comment]
Fixed on #286

**Comment by Pietro395** [comment]
@xeroticikot version 4.10.1 has been released
